src/components/JoinUs/Perks.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import breakpoint from 'styled-components-breakpoint'
+import { Row, Col } from '../grid'
+import { Padding } from 'styled-components-spacing'
+import { Paragraph, H2 } from '../Typography'
+import Image from '../Common/Image'
+
+import { Section, ShiftedColumn } from './elements'
+
+const PerksListContainer = styled(ShiftedColumn)`
+  display: grid;
+  row-gap: ${remcalc(24)};
+  grid-template-columns: 1fr 1fr;
+  column-gap: 42px;
+  padding-top: ${remcalc(24)};
+
+  ${breakpoint('smallPhone')`
+    grid-template-columns: 1fr 1fr;
+  `};
+
+  ${breakpoint('smallTablet')`
+    grid-template-columns: 1fr 1fr 1fr;
+  `};
+
+  ${breakpoint('tablet')`
+    grid-template-columns: 1fr 1fr;
+    row-gap: ${remcalc(36)};
+    column-gap: 42px;
+    padding-top: ${remcalc(12)};
+  `};
+`
+
+const Icon = styled(Image)`
+  width: 60px;
+  height: 60px;
+`
+
+const PerkContainer = styled(Padding)`
+  display: flex;
+  flex-direction: column;
+`
+
+const PerkDescription = styled(Paragraph)`
+  padding: ${remcalc(12)} 0;
+`
+const Perk = ({ icon, description }) => (
+  <PerkContainer>
+    <Icon image={icon} />
+    <PerkDescription noMargin>{description}</PerkDescription>
+  </PerkContainer>
+)
+
+const PerksList = ({ perks }) => (
+  <PerksListContainer width={[1, 1, 1, 1, 1, 5 / 12]}>
+    {perks.map((perk, idx) => (
+      <Perk
+        key={idx}
+        icon={perk.icon}
+        description={perk.description.description}
+      />
+    ))}
+  </PerksListContainer>
+)
+
+const Perks = ({ data: { title, text, perks } }) => (
+  <Section>
+    <Row>
+      <Col width={[1, 1, 1, 1, 1, 5 / 12]}>
+        <H2>{title}</H2>
+        <Paragraph muted>{text}</Paragraph>
```

Comment:
![image](https://user-images.githubusercontent.com/3236388/51606112-e85b2e80-1f08-11e9-8d1c-0206273abe67.png)

It varies, sometimes it's 23 other 20, probably 24px is a better value since it complies with our spacing system.

Reply:
then again, 8px is the default for the h2, I'll add the 12px on the body text